packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -678,15 +696,26 @@ function onCanvasClick() {
 	if (isCreatingNewEdge.value) {
 		cancelNewEdge();
 	}
+	hideMenuButton();
+}
+
+function hideMenuButton() {
+	if (!isMenufocused.value) {
+		isOutputMenuShown.value = false;
```

Comment:
Maybe we don't need, `isOutputMenuShown`. Instead you can set `nodeWithOpenedMenu` to be null here and just show the menu if `nodeWithOpenedMenu` exists